Zend/zend_language_parser.y

```diff
@@ -1295,6 +1298,8 @@ variable:
 			{ $$ = $1; }
 	|	array_object_dereferencable T_OBJECT_OPERATOR property_name
 			{ $$ = zend_ast_create(ZEND_AST_PROP, $1, $3); }
+	|	array_object_dereferencable T_NULLSAFE_OBJECT_OPERATOR property_name
+			{ $$ = zend_ast_create(ZEND_AST_NULLSAFE_PROP, $1, $3); }
```

Comment:
Maybe this can be moved to `expr`? This would make most usages of `?->` in write context parse error. It would require adding `?->` in some places (e.g. `encaps_var`) @nikic WDYT?

Reply:
I don't see how that could work. Wouldn't that force you to write `($a?->b)->c` etc?

Reply:
Yeah you're right.